pkg/cluster/internal/create/actions/postcreationconfiguration/postcreationconfiguration.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package postcreationconfiguration implements the action for configuring cluster after cluster creation
+// like labelling, tainting, etc the nodes.
```

Comment:
I'm going to delete the whole bad action abstraction when I get some uninterrupted development time anyhow, but until then, these should be scoped to specific tasks so we can easily skip specific tasks.